src/js/load-package.ts

```diff
@@ -282,11 +282,24 @@ async function loadDynlib(lib: string, shared: boolean) {
     byteArray = Module.FS.readFile(lib);
   }
   const releaseDynlibLock = await acquireDynlibLock();
+
+  // This is a fake FS-like object to make emscripten
+  // load shared libraries from the file system.
+  const libraryFS = {
+    _rootPath: lib.split("/").slice(0, -1).join("/"),
```

Comment:
Could you add a comment here explaining why this is the root path?

Reply:
Okay, I added a comment and renamed `_rootPath` to `_ldLibraryPath`, which seems more intuitive.
We are currently installing shared libraries into site-packages directory, probably in a subsequent PR, we should change it to install all shared libraries into `/usr/local/lib` or `/usr/lib`.

Reply:
Sounds good.